provider/thedoctor.py

```diff
@@ -7,7 +7,7 @@
 
 class TheDoctor (Thread):
     # maximum time to allow a consumer to not successfully poll() before restarting
-    poll_timeout_seconds = 2
+    poll_timeout_seconds = 20
```

Comment:
I noticed (anecdotally) that posting to a trigger URL can sometimes take over 5 seconds. The timeout value should be greater than that, plus allow any time for actually processing/parsing messages. Do you have some other value in mind?